src/main/webapp/jsp/instructorFeedbackCopyModal.jsp

```diff
@@ -14,23 +10,28 @@
            id="<%= Const.ParamsNames.COPIED_FEEDBACK_SESSION_NAME %>"
            type="text"
            name="<%= Const.ParamsNames.COPIED_FEEDBACK_SESSION_NAME %>"
-           value="<%= data.fsName %>">
+           value="${data.fsName}">
 </div>
-<% for (CourseAttributes course: data.courses) { %>
+
+<c:forEach items="${data.courses}" var="course">
     <div class="checkbox">
         <label>
             <input type="checkbox"
                    name="<%= Const.ParamsNames.COPIED_COURSES_ID %>"
-                   value="<%= course.id %>">
-            <% if (course.id.equals(data.courseId)) { %>
-                [<span class="text-color-red"><%= course.id %></span>] : <%= course.name %>
-                <br>
-                <span class="text-color-red small">{Session currently in this course}</span>       
-            <% } else { %>
-                [<%= course.id %>] : <%= course.name %>
-            <% } %>
+                   value="${course.id}">
+            <c:choose>
+                <c:when test="${course.id == data.courseId}">
```

Comment:
Is there a better way to do equality testing rather than like this? I can't think of any one else but this looks like a non-optimal way as it doesn't abstract out the logic.